modules/HomePage/components/TaskForm/TaskForm.tsx

```diff
@@ -3,10 +3,15 @@ import React, { useState } from "react";
 import { DateInput } from "@mantine/dates";
 import { Container, Select, TextInput, Button, Group } from "@mantine/core";
 import "@mantine/dates/styles.css";
+import { notifications } from "@mantine/notifications";
 
 import { Task } from "../../types/Task";
 import classes from "./TaskForm.module.css";
-import { useAddTodoMutation } from "@/services/todoApi";
+import {
+  useAddTodoMutation,
+  useDeleteCompletedTodosMutation,
+  useGetTodosQuery,
+} from "@/services/todoApi";
```

Comment:
@ imports should be above relative imports

Reply:
Addressed